functions/Get-DbaSqlManagementObject.ps1

```diff
@@ -0,0 +1,88 @@
+﻿function Get-DbaSqlManagementObject
+{
+<#
+	.SYNOPSIS
+	Gets SQL Mangaement Object versions installed on the machine.
+
+	.DESCRIPTION
+	 The Get-DbaSqlManagementObject returns an object with the Version and the Add-Type Load Template for each version on the server.
+
+	.PARAMETER VersionNumber
+	This is the specific version number you are looking for. The function will look 
+	for that version only.
+	
+	.PARAMETER Silent
+	Use this switch to disable any kind of verbose messages
+	
+	.NOTES
+	Original Author: Ben Miller (@DBAduck - http://dbaduck.com)
+
+	Tags: SMO
+
+	Website: https://dbatools.io
+	Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+	License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+	.LINK
+	https://dbatools.io/Get-DbaSqlManagementObject
+
+	.EXAMPLE
+	Get-DbaSqlManagementObject
+	Returns all versions of SMO on the computer
+	
+	.EXAMPLE
+	Get-DbaSqlManagementObject -VersionNumber 13
+	Returns just the version specified. If the version does not exist then it will return nothing.
+	
+
+#>
```

Comment:
I have fixed the things up like the command in the link. It looks much more readable and standard. Thanks.